docs/UserGuide.md

```diff
@@ -178,51 +178,66 @@ Precise expected outputs on failure:
 * Parameters: INDEX (must be a positive integer)."
 * is returned to the user.
 
-![edit format](images/cloneinvalidindex.png)
+![edit format](images/clone-UG/cloneinvalidindex.png)
 
 * When the index entered is greater than the current number of contacts in the address book, the error message
 * “The person index provided is invalid.”
 * is returned to the user.
 
-![edit format](images/clonelargeindex.png)
+![edit format](images/clone-UG/clonelargeindex.png)
 
 * When the person at index entered has already been cloned in FAPro, the error message
 * “A clone of this person already exists.
 * To clone again, please edit the previous clone first or alternatively, clone the previous clone.”
 * is returned to the user.
 
-![edit format](images/clonebeforeerror.png)
+![edit format](images/clone-UG/clonebeforeerror.png)
 
 ### Deleting a person : `delete`
 
-Deletes a contact from FAPro at the specified index.
+Deletes a contact from FAPro at the specified indexes.
 
-Format: `delete INDEX`
-* Deletes the person at the specified `INDEX`.
+Format: `delete INDEX1 INDEX2 INDEX 3`
+* Deletes the person at `INDEX1`, `INDEX2' and `INDEX3'. (i.e if delete 1 2 3 is input, deletes person at index 1, index 2 and index 3 respectively)
 * The index refers to the index number shown in the displayed person list.
 
 Acceptable parameters for INDEX:
-* The index **must be a positive integer** 1, 2, 3, …​ (less than the size of the contact list in FAPro)
+* The indexes **must be a positive integers** 1, 2, 3, …​ (less than the size of the contact list in FAPro)
+* The indexes must have a space inbetween and no commas (i.e delete 1,2,3 is invalid)
```

Comment:
I think in between have spaces?
and do you think we should add period for bullet points or nope? I think we should standardize this.

Reply:
Correct inbetween i have spaces but i was talking more about the individual indexes. I do get the confusion though. Will rectify on my end.